controllers/tools/views/applications.njk

```diff
@@ -161,7 +161,8 @@
                                                 label: '{{ country.title }}',
                                                 data: {{ country.data | dump(2) | safe }},
                                                 borderColor: '{{ country.colour }}',
-                                                fill: '{{ country.colour }}'
+                                                fill: '{{ country.colour }}',
```

Comment:
Do we still need `fill`?

Reply:
Possibly not but I left it in in case we change back and this way it'll still work with the line format.